node_modules/oae-authentication/lib/rest.js

```diff
@@ -44,3 +43,113 @@ require('./strategies/twitter/rest');
  */
 OAE.globalAdminRouter.on('post', '/api/auth/logout', AuthenticationAPI.logout);
 OAE.tenantRouter.on('post', '/api/auth/logout', AuthenticationAPI.logout);
+
+/**
+ * Add two endpoints for REST control
+ */
+
+
+var _getResetPasswordSecret = function(req, res){
+	AuthenticationAPI.getResetPasswordSecret(req.ctx, req.params.tenantAlias, req.params.username, function(err, token){
+	    if (err) {
+	        return callback(err);
+	    }
+
+		return res.send(token);
+	})
+};
+
+/**
+ * @REST Globalresetinit
+ *
+ * generate an user token based on a existing username
+ *
+ * @Server      Admin
+ * @Method      GET
+ * @Path        /auth/local/reset/init/{tenatAlias}/{username}
```

Comment:
In hindsight, it might make more sense to have /auth/local/reset/init/{userId} on the global admin. Global administrators won't have a user's username readily available